app/graphql/mutations/integration_items/fetch_items.rb

```diff
@@ -20,6 +20,8 @@ def resolve(**args)
 
         ::Integrations::Aggregator::SyncService.call(integration:)
 
+        integration.integration_items.where(item_type: :standard).destroy_all
```

Comment:
Should it be part of the sync service?

Reply:
Actually sync service should just trigger various syncs on Nango side. I moved this line to the items service where items are actually fetched